web/src/components/molecules/Schema/ModelCreationModal/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useCallback } from "react";
+
+import Form from "@reearth-cms/components/atoms/Form";
+import Input from "@reearth-cms/components/atoms/Input";
+import Modal from "@reearth-cms/components/atoms/Modal";
+import TextArea from "@reearth-cms/components/atoms/TextArea";
+
+export interface FormValues {
+  name: string;
+  description: string;
+  key: string;
+}
+
+export interface Props {
+  open?: boolean;
+  onClose?: (refetch?: boolean) => void;
+  onSubmit?: (values: FormValues) => Promise<void> | void;
+}
+
+const initialValues: FormValues = {
+  name: "",
+  description: "",
+  key: "",
+};
+
+const ModelCreationModal: React.FC<Props> = ({ open, onClose, onSubmit }) => {
+  const [form] = Form.useForm();
+
+  const handleSubmit = useCallback(() => {
+    form
+      .validateFields()
+      .then(async values => {
+        await onSubmit?.(values);
+        onClose?.(true);
+        form.resetFields();
+      })
+      .catch(info => {
+        console.log("Validate Failed:", info);
```

Comment:
Has @lavalse made any designs for an error notification or error modal to handle something like this? Or we don't need?